Core/Relationships/RelationshipResolver.cs

```diff
@@ -120,20 +120,44 @@ public RelationshipResolver(RelationshipResolverOptions options, IRegistryQuerie
         /// <param name="registry"></param>
         /// <param name="kspversion"></param>
         public RelationshipResolver(IEnumerable<string> module_names, RelationshipResolverOptions options, IRegistryQuerier registry,
-            KSPVersion kspversion) :
-                this(module_names.Select(name => CkanModule.FromIDandVersion(registry, name, kspversion)).ToList(),
-                    options,
-                    registry,
-                    kspversion)
+            KSPVersion kspversion) : this(options,registry, kspversion)
         {
-            // Does nothing, just calls the other overloaded constructor
+            List<Module> modules = new List<Module>();
+            foreach (string identifier in module_names)
+            {
+                try
+                {
+                    modules.Add(CkanModule.FromIDandVersion(registry, identifier, kspversion));
+                }
+                catch (ModuleNotFoundKraken kraken)
+                {
+                    // Let's see if the mod was manually installed (via a .ckan, for example)
+                    InstalledModule installed = registry.InstalledModule(identifier);
+                    if (installed != null)
+                    {
+                        modules.Add(installed.Module);
+                    }
+                    else
+                    {
+                        throw kraken;
```

Comment:
Can we get a kraken with a more verbose message by any chance? If this ever ends up in a bug report it would be great to have a message indicating what happened. :)